raiden-dapp/src/locales/en.json

```diff
@@ -529,8 +529,12 @@
     "select-button": "Select Hub",
     "service-token-tooltip": "Mint and deposit {token} to the User Deposit Contract to pay for the Monitoring Service.",
     "service-token-tooltip-main": "Deposit {token} to the User Deposit Contract to pay for the Monitoring Service.",
-    "service-token-balance-too-low": "Your {token} balance is too low to pay for the Monitoring Service. Use the button above to mint and deposit more tokens.",
-    "service-token-description": "Balance to pay the Monitoring Service"
+    "service-token-description": "BALANCE FOR MONITORING SERVICE"
```

Comment:
why is this all caps?

Reply:
Dunno, designers decision 😉

Reply:
Shouldn't we then use something like `text-transform: uppercase`?

Reply:
Ah, yes, of course, that makes more sense.